notebooks/inference_notebook.ipynb

```diff
@@ -0,0 +1,287 @@
+{
```

Comment:
<p>Why is this notebook called "inference_notebook"? and not "data_collection" or something like that? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/operate-first-data-science-community/pull/59/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Yes, data-collection makes more sense. May be word `inference` would create a confusion. I will change the name now. Thanks.